src/OneExplorer/OneExplorer.ts

```diff
@@ -621,7 +648,7 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
     let approval: string;
     let useTrash: boolean;
 
-    if (vscode.env.remoteName) {
+    if (this.isRemote) {
```

Comment:
Maybe...?

```suggestion
    if (this.isRemote()) {
```

Reply:
It's a getter, so it doesn't need parenthesis :-D

https://www.typescripttutorial.net/typescript-tutorial/typescript-getters-setters/